cs481-ekh/f21-welolo
# Created and Wired Up Dummy Merchant Page (#43)

Verification steps:
- Start both servers
- When the react application opens, a merchant page will appear
- The price will transfer over to the payment page when you select the item using the button and it will not allow you to edit the amount field on the payment page.
- If you select the skip button at the bottom you will be asked to enter in a price on the payment page
